ql/src/test/queries/clientpositive/avro_timestamp_micros.q

```diff
@@ -0,0 +1,3 @@
+CREATE EXTERNAL TABLE hive_test(`dt` timestamp) STORED AS AVRO;
```

Comment:
This needs to have more test cases. Consider atleast referring to existing test case from millis and extend it for micros.
We are handling a lot of cases, such as proleptic calendar, zone based conversions etc.

Reply:
Understandable, adding equivalent test cases to avro in milliseconds precision, and some more test classes as required